.husky/pre-commit

```diff
@@ -0,0 +1,3 @@
+cd frontend
+npx lint-staged
+git add .
```

Comment:
This would add any untracked changes too. 
I think the pre-commit hook should just check, rather than update and add files?

Reply:
Changed it to just check so users will have to format themselves